serialization/EasyCaching.Serialization.MemoryPack/DefaultMemoryPackSerializer.cs

```diff
@@ -0,0 +1,33 @@
+using EasyCaching.Core.Serialization;
+using MemoryPack;
+
+namespace EasyCaching.Serialization.MemoryPack;
+
+/// <summary>
+/// Default MemoryPack serializer
+/// </summary>
+public class DefaultMemoryPackSerializer : IEasyCachingSerializer
+{
+    private readonly string _name;
+    private readonly MemoryPackSerializerOptions _memoryPackSerializerOptions;
+
+    public string Name => _name;
+
+    public DefaultMemoryPackSerializer(string name, MemoryPackSerializerOptions options = null)
+    {
+        _name = name;
+        _memoryPackSerializerOptions = options;
+    }
+
+    public T Deserialize<T>(byte[] bytes) => MemoryPackSerializer.Deserialize<T>(bytes, _memoryPackSerializerOptions);
+    public object Deserialize(byte[] bytes, Type type) => MemoryPackSerializer.Deserialize(type, bytes, _memoryPackSerializerOptions);
+    public object DeserializeObject(ArraySegment<byte> value) => throw new NotImplementedException("this is not supported in MemoryPack serializer");
```

Comment:
MemoryPack don't support this one?

Reply:
No it does not, it needs the type to be deserialized to. It can't handle dynamic or non memoryPackable objects.